src/com/amazon/ion/benchmark/GenerateAndOrganizeBenchmarkResults.java

```diff
@@ -0,0 +1,71 @@
+package com.amazon.ion.benchmark;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class GenerateAndOrganizeBenchmarkResults {
+
+    final public static List<String> BENCHMARK_OPTION_COMBINATIONS = Arrays.asList(
+        "java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar read --api dom --results-format ion ",
+        "java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar read --api streaming --results-format ion ",
+        "java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar write --api streaming --results-format ion ",
+        "java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar read --api streaming --format json --results-format ion ",
+        "java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar read --api dom --format json --results-format ion ",
+        "java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar write --api streaming --format json --results-format ion ");
```

Comment:
> It would be great if we could retrieve the version (in this case `0.0.1`) from a centralized location so we don't have to change this code in the event that the CLI version changes. This may be complicated to do since the version number is declared in pom.xml, so at the very least we should store the version string in a constant so that if it does need to be changed in code it can be done on one line. We may also want to store the entire jar invocation in a constant (`java -jar target/...jar`) so that it can be reused here and below in `generateAndSaveBenchMarkResults`.

This can be achieved by parsing pom.xml as long as the position of version information in pom.xml will not be changed. Pom parser will be added in the following commit.